style.css

```diff
@@ -8,15 +8,28 @@
             width: 100%;
         }
         
-        .menu {
-            width: 80%;
+        .part{
+            width:100%;
+        }
+
+        .limit{
+            width:80%;
             margin: 0 auto;
+            box-sizing: border-box;
         }
-        
+
+        .menu {
+            position: fixed;
+            top:0;left:0;
+            z-index: 999;
+            background-color: white;
+        }
+
+
         #mleft {
             font-style: italic;
-            width: 30em;
-            float: left
+            float: left;
+            padding: 1em;
         }
```

Comment:
Usunąłem width: 30em. Tworzyło to sztuczną szerokość w prawo i przy zmniejszeniu szerokości ekranu od razu spychało prawą część menu pod spód. A tak to szerokość dopasowała się do napisu. Padding dodałem po to, żeby był odstęp od góry i dołu.